app/app.js

```diff
@@ -198,10 +198,11 @@ export default class App {
         const username = `${deviceToken}, ${currentUserId}`;
         const password = `${token},${url}`;
 
+        this.token = token;
```

Comment:
Why is this moving?

Reply:
with the explanation below one of the things that happen is that we need to detect if you are already logged in or not to send you to the select server screen or the channel screen bypassing the entry screen that does not make sense to load at that point.

When you open the app the first time and then login the app.token and app.url are not set as they are handled by the login action, with this we are ensuring that they are set at that moment, so if you decide to split the screen it will know what screen to initialize.